boot/zephyr/main.c

```diff
@@ -363,6 +363,19 @@ void main(void)
     }
 #endif
 
+    rc = boot_go(&rsp);
+    if (rc != 0) {
+        BOOT_LOG_ERR("Unable to find bootable image");
+        while (1)
+            ;
+    }
+
+    BOOT_LOG_INF("Bootloader chainload address offset: 0x%x",
+                 rsp.br_image_off);
+
+    BOOT_LOG_INF("Jumping to the first image slot");
+    ZEPHYR_BOOT_LOG_STOP();
```

Comment:
Should BOOT_LOG_STOP be moved back? So that fprotect_area fail will print it's message?

Reply:
moved it further down